lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1469,8 +1470,7 @@ void LocalEnforcer::handle_set_session_rules(
 
       // Propagate these rule changes to PipelineD and MME (if 4G)
       propagate_rule_updates_to_pipelined(
-          session->get_session_id(), config, pending_activation,
-          pending_deactivation, false);
```

Comment:
it looks like i was passing a sting directly not a reference. Not sure why it was not crashing